packages/wonder-blocks-tooltip/components/tooltip-popper.js

```diff
@@ -37,14 +29,43 @@ type Props = {|
 |};
 
 export default class TooltipPopper extends React.Component<Props> {
-    _renderPositionedContent(childrenProps: PopperJSChildrenProps) {
-        const {children, placement} = this.props;
+    _bubbleRefTracker = new RefTracker();
+    _tailRefTracker = new RefTracker();
+
+    _getUpdatedBubbleRefTracker(targetRefFn: (?HTMLElement) => void) {
+        this._bubbleRefTracker.setCallback(targetRefFn);
+        return this._bubbleRefTracker.updateRef;
+    }
+
+    _getUpdatedTailRefTracker(targetRefFn: (?HTMLElement) => void) {
```

Comment:
Where it this called?

Reply:
Left over from interim solution. I'll delete. Thanks!